test/end-to-end/test/test_importSanity.js

```diff
@@ -4,10 +4,20 @@ const EditRecipePage = require('../pages/editRecipe');
 const apiCall = require('../utils/apiCall');
 const pageConfig = require('../config');
 const sqlite3 = require('sqlite3').verbose();
+const fs = require('fs');
 
 describe('Imported Content Sanity Testing', function () {
   this.timeout(15000);
-  const db = new sqlite3.Database(process.env.MDDB || 'metadata.db');
+  let db;
+
+  before((done) => {
+    // Check exist of metadata.db file first
+    fs.access(process.env.MDDB || 'metadata.db', (error) => {
+      if (error) return done(error);
```

Comment:
Where does the error message come from? Is it a 'File not found' type of error ?